src/components/Navbar.vue

```diff
@@ -14,7 +13,7 @@
           aria-expanded="false"
         >
           <span class="badge badge-pill badge-light">{{cartQty}}</span>
-          <i class="fas fa-shopping-cart mx-2"></i>
+          <font-awesome-icon icon="shopping-cart" class="mx-2"></font-awesome-icon>
```

Comment:
This looks good. Nothing wrong with this, but could be written as `<font-awesome-icon icon="shopping-cart" class="mx-2" />` I guess. Personal preference.